README.md

```diff
@@ -2,109 +2,116 @@
  <img src="https://docs.freetubeapp.io/images/logoColor.png" width=500 align="center">
 </p>
 
-FreeTube is an open source desktop YouTube player built with privacy in mind.
-Use YouTube without advertisements and prevent Google from tracking you with their cookies and JavaScript.
-Available for Windows, Mac & Linux thanks to Electron.
+**Copylefted libre software desktop YouTube player** (with privacy).
 
-Please note that FreeTube is currently in Beta. While it should work well for
-most users, there are still bugs and missing features that need to be
-addressed.
+No advertisements or local Google claws tracking you with cookies and JavaScript.
```

Comment:
This is extremely subjective, but the use of "claws" as a noun and the lack of a verb in this sentence made it hard for my brain to parse this sentence.